lms/services/canvas_api/client.py

```diff
@@ -225,6 +247,7 @@ class _ListFilesSchema(RequestsResponseSchema):
         display_name = fields.Str(required=True)
         id = fields.Integer(required=True)
         updated_at = fields.String(required=True)
```

Comment:
This is the change to the schema to mean we get the size too. This forces a change in seemingly unrelated tests